.github/workflows/unit-tests.yml

```diff
@@ -238,12 +240,131 @@ jobs:
+          7z x php_apcu-5.1.22-8.2-ts-vs16-x86.zip -y >nul
+          iwr -outf php_redis-6.0.0-dev-8.2-ts-vs16-x86.zip https://github.com/symfony/binary-utils/releases/download/v0.1/php_redis-6.0.0-dev-8.2-ts-vs16-x86.zip
+          7z x php_redis-6.0.0-dev-8.2-ts-vs16-x86.zip -y >nul
+          cd ..
+          Copy php.ini-development php.ini-min
+          "memory_limit=-1" >> php.ini-min
+          "serialize_precision=-1" >> php.ini-min
+          "max_execution_time=1200" >> php.ini-min
+          "post_max_size=2047M" >> php.ini-min
+          "upload_max_filesize=2047M" >> php.ini-min
+          "date.timezone=`"America/Los_Angeles`"" >> php.ini-min
+          "extension_dir=ext" >> php.ini-min
+          "extension=php_xsl.dll" >> php.ini-min
+          "extension=php_mbstring.dll" >> php.ini-min
+          Copy php.ini-min php.ini-max
+          "zend_extension=php_opcache.dll" >> php.ini-max
+          "opcache.enable_cli=1" >> php.ini-max
+          "extension=php_openssl.dll" >> php.ini-max
+          "extension=php_apcu.dll" >> php.ini-max
+          "extension=php_igbinary.dll" >> php.ini-max
+          "extension=php_redis.dll" >> php.ini-max
+          "apc.enable_cli=1" >> php.ini-max
+          "extension=php_intl.dll" >> php.ini-max
+          "extension=php_fileinfo.dll" >> php.ini-max
+          "extension=php_pdo_sqlite.dll" >> php.ini-max
+          "extension=php_curl.dll" >> php.ini-max
+          "extension=php_sodium.dll" >> php.ini-max
+          Copy php.ini-max php.ini
+          cd D:\a\symfony\symfony
+          iwr -outf composer.phar https://getcomposer.org/download/latest-stable/composer.phar
+          mkdir %APPDATA%\Composer && Copy .github\composer-config.json %APPDATA%\Composer\config.json
+
+      - name: Install dependencies
+        id: setup
+        run: |
+          SET PATH=c:\php;%PATH%
+          mkdir %APPDATA%\Composer && copy /Y .github\composer-config.json %APPDATA%\Composer\config.json
+          git config --global user.email ""
+          git config --global user.name "Symfony"
+          FOR /F "tokens=* USEBACKQ" %%F IN (`bash -c "grep ' VERSION = ' src/Symfony/Component/HttpKernel/Kernel.php | grep -o '[0-9][0-9]*\.[0-9]'"`) DO (SET SYMFONY_VERSION=%%F)
+          php .github/build-packages.php HEAD^ %SYMFONY_VERSION% src\Symfony\Bridge\PhpUnit
+          SET COMPOSER_ROOT_VERSION=%SYMFONY_VERSION%.x-dev
+
+          echo ::group::composer update
+          php composer.phar update --no-progress --ansi
+          echo ::endgroup::
+
+          echo ::group::phpunit install
+          php phpunit install
+          echo ::endgroup::
+
+          echo ::group::memurai-developer install
+          choco install --no-progress memurai-developer
+          echo ::endgroup::
+
+      - name: Run tests (minimal extensions)
+        if: always() && steps.setup.outcome == 'success'
+        run: |
+          $env:Path = 'c:\php;' + $env:Path
```

Comment:
isn't this missing `shell: pwsh` (as you configure the default shell to be `cmd` ? This looks like a powershell syntax

Reply:
Yep, I was switching back and forth between both shells and missed this part. Fixed now!